examples/accessors_generator/main.py

```diff
@@ -0,0 +1,126 @@
+    # Function to retrieve the field name.
+    name: str = field.name.capitalize()
+
+    if field.associated_comment:
+        for text in get_devana_comments(field.associated_comment):
+            if text.startswith("devana: custom-name="):
+                name = text.split("=")[1]
+                break
+
+    return name
+
+
+def create_field_methods(field: FieldInfo) -> Tuple[MethodInfo, MethodInfo]:
+    # Function to generate setter and getter for field.
+    def should_ignore_attributes(comment: Optional[Comment]) -> bool:
+        return "devana: ignore-attributes" in get_devana_comments(comment) if comment else False
+
+    field_name: str = get_field_name(field)
+    maybe_unused_attribute: Attribute = Attribute("maybe_unused")
+    nodiscard_attribute: Attribute = Attribute("nodiscard")
+
+    field_setter: MethodInfo = MethodInfo()
+    field_setter.body = f"{field.name} = new{field.name.capitalize()};"
+    field_setter.access_specifier = AccessSpecifier.PUBLIC
+    field_setter.name = f"set{field_name}"
+    field_setter.return_type = BasicType.VOID
+
+    setter_arg: FunctionInfo.Argument = FunctionInfo.Argument()
+    setter_arg.name = f"new{field.name.capitalize()}"
+    setter_arg.type.details = field.type
+    if field.type.name == "string":
+        setter_arg.type.modification |= TypeModification.CONST | TypeModification.REFERENCE
+
+    field_setter.arguments.append(setter_arg)
+
+    field_getter: MethodInfo = MethodInfo()
+    field_getter.body = f"return {field.name};"
+    field_getter.access_specifier = AccessSpecifier.PUBLIC
+    field_getter.name = f"get{field_name}"
+    field_getter.return_type = field.type
+    field_getter.modification |= FunctionModification.CONST
+
+    if not should_ignore_attributes(field.associated_comment):
+        field_getter.attributes.append(
+            AttributeDeclaration([nodiscard_attribute, maybe_unused_attribute])
+        )
+        field_setter.attributes.append(
+            AttributeDeclaration([maybe_unused_attribute])
+        )
+
+    return field_setter, field_getter
+
+
+def create_class_methods(class_info: ClassInfo):
+    # Function to generate methods for all class fields.
+    def should_ignore_field(comment: Optional[Comment]) -> bool:
+        return "devana: ignore-field" in get_devana_comments(comment) if comment else False
+
+    for constructor in filter(lambda constr: isinstance(constr, ConstructorInfo), class_info.constructors):
+        if constructor.body == "{}":
+            # We need to define body (as empty in this case) to say devana: it is function definition.
+            constructor.body = ""
+
+    for private_field in filter(lambda field: isinstance(field, FieldInfo), class_info.private):
+        if should_ignore_field(private_field.associated_comment):
+            continue
+
+        class_info.content.extend(create_field_methods(field=private_field))
+
+
+def load_files_content() -> Iterable[Tuple[str, Iterable[ClassInfo]]]:
```

Comment:
Please make from this named tuple or even better  dataclass.
> Iterable[Tuple[str, Iterable[ClassInfo]]]

It is the same horrible code in python like in c++ templates:

> Iterable<Tuple<str, Iterable<ClassInfo>>>>